tests/unit/fem/test_fe.cpp

```diff
@@ -676,3 +676,52 @@ TEST_CASE("L2 Wedge Finite Element",
       }
    }
 }
+
+TEST_CASE("Nedelec Wedge Finite Element",
+          "[ND_WedgeElement]"
+          "[VectorFiniteElement]"
+          "[FiniteElement]")
+{
+   int p = 1;
+
+   ND_WedgeElement fe(p);
+
+   SECTION("Attributes")
+   {
+      REQUIRE( fe.GetDim()            == 3                     );
+      REQUIRE( fe.GetGeomType()       == Geometry::PRISM       );
+      REQUIRE( fe.GetDof()            == 3*p*(p+1)*(p+2)/2     );
+      REQUIRE( fe.GetOrder()          == p                     );
+      REQUIRE( fe.Space()             == (int) FunctionSpace::Qk     );
+      REQUIRE( fe.GetRangeType()      == (int) FiniteElement::VECTOR );
+      REQUIRE( fe.GetMapType()        == (int) FiniteElement::H_CURL );
+      REQUIRE( fe.GetDerivType()      == (int) FiniteElement::CURL   );
+      REQUIRE( fe.GetDerivRangeType() == (int) FiniteElement::VECTOR );
+      REQUIRE( fe.GetDerivMapType()   == (int) FiniteElement::H_DIV  );
+   }
+}
+
+TEST_CASE("Raviart-Thomas Wedge Finite Element",
+          "[RT_WedgeElement]"
+          "[VectorFiniteElement]"
+          "[FiniteElement]")
+{
+   int p = 1;
+
+   RT_WedgeElement fe(p-1);
+
+   SECTION("Attributes")
+   {
+      REQUIRE( fe.GetDim()            == 3                       );
+      REQUIRE( fe.GetGeomType()       == Geometry::PRISM         );
+      REQUIRE( fe.GetDof()            == (int)pow(p+1,2)*p/2 +
+               /*                     */ (int)pow(p,2)*(p+2)     );
```

Comment:
Is this comment here to make the styler behave?

Reply:
Yes, that's all it is.